src/requests.ts

```diff
@@ -51,12 +63,15 @@ export class RequestHelper {
         return data;
     }
 
-    public async getRoverPhotos(): Promise<RoverPhotos> {
-        const roverPhotoResponse = await axios.get('https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos?sol=0&camera=FHAZ&api_key=' + this.apiKey);
-        console.log(roverPhotoResponse.data)
-        let data = roverPhotoResponse.data;
-
-        return data;
+    public async getRoverPhotos(roverName: string, cameraType: cameraTypes): Promise<RoverPhotos> {
+        try {
+            const roverPhotoResponse = await axios.get(`https://api.nasa.gov/mars-photos/api/v1/rovers/${roverName}/photos?sol=0&camera=${cameraType}&api_key=${this.apiKey}`)

```

Comment:
it might be worth printing out what string is created when you have `https://api.nasa.gov/mars-photos/api/v1/rovers/${roverName}/photos?sol=0&camera=${cameraType}&api_key=${this.apiKey}`.

I have a feeling `cameraType` enum value is actually just an integer. 

You could either make the enum a string type like
```
enum CameraType {
  FHAZ = 'FHAZ',
  ...
}
```

or make a "string union type" like
```
type CameraType = 'FHAZ' | 'RHAZ' | ...
```


There are lots of controversy around Typescript enum due to this confusing behaviour. I think it's generally recommended to use the string union type.